src/elements/emby-slider/emby-slider.js

```diff
@@ -130,6 +137,57 @@ import '../emby-input/emby-input';
         });
     }
 
+    function setChapterMark(range, valueChapterMark, chapterMark, valueProgress) {
+        requestAnimationFrame(function () {
+            const bubbleTrackRect = range.sliderBubbleTrack.getBoundingClientRect();
+            const chapterMarkRect = chapterMark.getBoundingClientRect();
+
+            if (!bubbleTrackRect.width || !chapterMarkRect.width) {
+                // width is not set, most probably because the OSD is currently hidden
+                return;
+            }
+
+            let chapterMarkPos = (bubbleTrackRect.width * valueChapterMark / 100) - chapterMarkRect.width / 2;
+            chapterMarkPos = Math.min(Math.max(chapterMarkPos, - chapterMarkRect.width / 2), bubbleTrackRect.width - chapterMarkRect.width / 2);
```

Comment:
Well, the early return if the width is 0 was just meant to prevent recalculating the marker positions even if the OSD is hidden. This lead to noticeable jumps of the markers when the OSD is shown again and the width is correct again.

About the resizing: Could you elaborate?
Do you mean if the positioning can handle a resize during playback? If so, then yes, as long as updateValues is continued to be called.